### DISCUSSION TITLE
Update availability for vendor items

### DISCUSSION THREAD

**Discussion by PerryWolf** [amzn/selling-partner-api-models#3735] [General] [answered]
Hello everyone,

is there any way to update the availability of items of a vendor account via SP-API?
I have already tried using the Listings Feed API (JSON_LISTINGS_FEED), but had no success.
It also seems to me that this API cannot be used for this.
I can upload an XLSX file with availabilities via Vendor Central.
The file can be used to choose between 3 statuses (Available, Temporarily not available, Permanently not available), while I have to enter an exact quantity for the JSON_LISTINGS_FEED (/attributes/fulfillment_availability).
So is it even possible to adjust the availability for these items via API?

**Comment by chapmanjw** [comment, marked as answer]
Vendor availability is handled separately from the product attributes and cost price handles as part of listings data. Currently, this is only available via Vendor Central. We do not have any current plans to address this in the Listings Items API (or JSON_LISTINGS_FEED), but I will share this feedback with the product management team for the Listings Items API.
